simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/IrvingFactsChange.scala

```diff
@@ -0,0 +1,89 @@
+package com.puppetlabs.gatling.node_simulations
```

Comment:
is there a reason this is a new file? is there a reason to not update an existing simulation file?  
also, the other simulation files are titled with the pe version number, not codename.  curious if there is a reason for breaking this pattern.

Reply:
If we are not updating an existing simulation file in this check-in, we at least need another ticket to update the ApplesToApples simulation.

Reply:
Eventually, I'd think we would want the "recording" script to make these modifications automagically. Before that happens, I think we should figure out a safe method to turn fact randomization off, so that other teams won't hesitate to use the integrated code.